sdk/nodejs/storage/v1beta1/CSINode.ts

```diff
@@ -8,16 +8,16 @@ import * as outputs from "../../types/output";
 import { getVersion } from "../../version";
 
     /**
-     * @deprecated storage/v1beta1/CSINode is deprecated by storage/v1beta1/CSINode and not
-     * supported by Kubernetes v<nil>+ clusters.
-     * 
      * CSINode holds information about all CSI drivers installed on a node. CSI drivers do not need
      * to create the CSINode object directly. As long as they use the node-driver-registrar sidecar
      * container, the kubelet will automatically populate the CSINode object for the CSI driver as
      * part of kubelet plugin registration. CSINode has the same name as a node. If the object is
      * missing, it means either there are no CSI Drivers available on the node, or the Kubelet
      * version is low enough that it doesn't create this object. CSINode has an OwnerReference that
      * points to the corresponding node object.
+     * 
+     * @deprecated storage/v1beta1/CSINode is deprecated by storage/v1beta1/CSINode and not
+     * supported by Kubernetes v<nil>+ clusters.
```

Comment:
This is incorrect, but not related to the PR specifically. I opened https://github.com/pulumi/pulumi-kubernetes/issues/943 to track